### PR TITLE
Fix potential instability in cacheIMSRange

### PR DESCRIPTION
**Pull request by macisasandwich**
Microserver may terminate early.

### CODE REVIEW

**Comment by dragon512** [comment]
[approve ci]

File: tests/gold_tests/headers/cachedIMSRange.test.py [hunk 1]
```diff
@@ -148,11 +156,13 @@
...
 # Test 9 - Fetch the 0 byte etag object again when fresh in cache to ensure its still a 0 byte object
 tr = Test.AddTestRun()
 tr.DelayStart=2
 tr.Processes.Default.Command = 'curl -s -D - -v --ipv4 --http1.1 -H"UID: EtagError" -H "x-debug: x-cache,x-cache-key,via" -H "Host: www.example.com" http://localhost:{0}/etag'.format(ts.Variables.port)
 tr.Processes.Default.ReturnCode = 0
 tr.Processes.Default.Streams.stdout = "cache_and_error_nobody.gold"
-tr.StillRunningAfter = ts
\ No newline at end of file
+tr.StillRunningAfter = ts
+tr.StillRunningAfter = server
```
Comment (duke8253) [inline tests/gold_tests/headers/cachedIMSRange.test.py]: Newline maybe?

Comment (macisasandwich) [reply tests/gold_tests/headers/cachedIMSRange.test.py]: Done.

**Comment by shinrich** [comment]
[approve ci]

**Pull request merged**